tutorials/scripting/gdextension/gdextension_docs_system.rst

```diff
@@ -172,3 +172,30 @@ Currently they supported tags for the GDExtension documentation system are:
         ``#ff00ff``, see :ref:`doc_bbcode_in_richtextlabel_hex_colors`).
 
     - ``[color={code/name}]{text}[/color]``
+
+
+Publishing documentation online
+-------------------------------
+
+You may want to publish an online reference for your GDExtension, akin to this website.
+The most important step is to build reStructuredText (``.rst``) files from your XML
+class reference:
+
+.. code-block:: none
+
+    # You'll need to have Python installed for this command to work.
+    curl -sSL https://raw.githubusercontent.com/godotengine/godot/master/doc/tools/make_rst.py | python3 - -o "docs/classes" -l "en" doc_classes
+
+Your ``.rst`` files will now be available in ``docs/classes/``. From here, you can use
+any documentation builder that supports reStructuredText syntax to create a website from them.
+`godot-docs <https://hosted.weblate.org/projects/godot-engine/godot-docs>`_ uses `Sphinx <https://www.sphinx-doc.org/en/master/>`_.
+You can use the repository as a basis to build your own documentation system.
+This is where you'll have to get a bit creative, but here's a rough outline
+of the steps involved:
```

Comment:
I think this should be phrased differently. Many of the people reading the docs will try to mimic the instructions even with a different setup so this should be phrased that the setup should be identical so the steps work. Otherwise we will get open issues complaining that the steps don't work.

Reply:
I'm trying to convey that the steps are very 'unprecise' with this sentence, because I needed to use a good amount of insight beyond this simple plan to actually get it to run. Describing all of the steps here would result in a pretty large and probably unhelpful and soon to be outdated tutorial. 

The only assumption the text is making is that the documentation files are in `doc_classes` (ensured by `doctool`). It should really work with any kind of setup beyond that.

Reply:
@Ivorforce Perhaps along the lines of: 
> The setup may differ depending on how your project is structured. Here are the exemplary steps you would need to take to compile the documentation into rst files.

Reply:
I think i see where you're coming from, but the following steps actually do not depend on the project structure at all. This is just a guideline for how to adapt the Godot docs with different content. It's basically independent of the project itself, at least after the XML Generation step.

Reply:
I have edited the section with a new phrasing. Hopefully this conveys it better?